portfolio-webapp/src/App.css

```
@@ -8,19 +10,39 @@ input,
 label {
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: 'DM Sans', sans-serif;
   box-sizing: border-box;
 }
 
 body {
   background-color: #f4f7f6;
+  font-family: 'DM Sans', sans-serif;
+  
+}
+
+.pattern-background {
+  overflow: auto;
+}
+.pattern-background::before {
+  content: "";
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 250px;
+  background: url('/wave.png');
```

This is pointing to a non existent file- perhaps it is cached on your device?
```suggestion
  background: url("../public/wave.png");
```

---

yeah i see why i worked on my end, I decided to move it to src/assets we use webpack can you check again if it works?